.travis.yml

```diff
@@ -1,16 +1,22 @@
+dist: xenial
 language: python
 sudo: false
 cache: pip
-python: 3.6
+python:
+  - "3.6"
+  - "3.7"
+  - "3.8"
 env:
   matrix:
   - DJANGO=111
   - DJANGO=20
+  - DJANGO=21
```

Comment:
Actually, can you rather replace 2.0 with 2.2 and drop 2.1. The only two versions with official support are 1.11 and 2.2 (and soon 3.0).
I just don't see a point in supporting versions that have reached EoL.